analyses/0014-vitess/vitess-analysis.md

```diff
@@ -0,0 +1,1019 @@
+
+Yes, documentation is versioned and prominently includes the latest Stable and
+Development versions.
+
+#### New user content
+
+**Getting started entry**: Is “getting started” clearly labeled?
+(“Getting started”, “Installation”, “First steps”, etc.)
+
+Yes, the Get Started section contains instructions to install, set up, and run
+Vitess in four different environments (OSes and container solutions).
+
+The Get Started guide has better written instructions than the User Guide, but
+procedures could still be clearer. There is no meta-text explaining when you
+would want to use each install option (the test installs are based on platform,
+but a discussion of development, test, and production environments would still
+help).
+
+**Installation**: Is installation documented step-by-step?
+
+No, instructions are not documented step-by-step, strictly speaking.
+Instructions are generally sequential down the page but not numbered. As with
+the User Guide procedures, steps could be rewritten so that they're more
+explicit.
+
+**Multiple OS**: If needed, are multiple OSes documented?
+
+Yes, multiple OSes (and container platforms) are documented. Instructions are
+given for a Kubernetes (production) install, and three local installs: Linux,
+Mac, and Docker. The local (non-Kubernetes) installs are "for test purposes,"
+but no further details are provided. The default local install for pre-compiled
+binaries does not explicitly list what OSes are supported.
+
+**Getting started next steps**: Do users know where to go after reading the
+getting started guide?
+
+Yes, there is a Next Steps section at the end of each install except the Docker
+local install. However, the Next Steps section is perfunctory, pointing in all
+cases to the Move Tables page in the Migration guide.
+
+**New user signpost**: Is your new user content clearly signposted on your site’s
+homepage or at the top of your information architecture?
+
+No, there is no clear entry path for new users. They'll probably make their way through
+the install and then try to figure out how to implement their migration case (which
+probably fits one of the documented scenarios, but there is no clear way for them
+to find it).
+
+**Sample code**: Is there easily copy-pastable sample code or other example content?
+
+Yes, command-line examples are plentiful. There is no click-to-copy, however. Users
+must highlight and copy manually.
+
+#### Content maintainability & site mechanics
+
+**Searchability**: Is your documentation searchable?
+
+Yes, there is a text Search feature that is limited to the currently displayed version.
+It seems to work very well.
+
+Search is full-text, so common search terms can result in an unwieldy number of results.
+
+**Localization & i18n directories**: Are you planning for localization/internationalization
+with regard to site directory structure?
+
+Yes, there are full versions of the documentation in both English and Chinese.
```

Comment:
We have been debating removing the Chinese documentation because it is being simply copied over for each new release without any content updates.

Reply:
@deepthi You mean you're copying a down-level version of the documentation without the updates for new releases? Which version of the documentation is the current Chinese version?

Reply:
> I would recommend keeping it, but maybe stop versioning it, or suggest it's archived or deprecated somehow.

I'd recommend removing it from any release later than the last time it was updated, and redirect from the later releases to that version.